podpac/core/data/types.py

```diff
@@ -48,6 +49,8 @@
 h5py = lazy_module('h5py')
 boto3 = lazy_module('boto3')
 requests = lazy_module('requests')
+zarr = lazy_module('zarr')
+s3fs = lazy_module('s3fs')
```

Comment:
@mpu-creare Should I add these to the setup.py under optional imports somewhere?

Reply:
Answered this myself.

Reply:
I agree with what you did, FWIW.